### PR TITLE
Fix collapseto

### PR DESCRIPTION
**Pull request by GiggleLiu**

### CODE REVIEW

**Comment by Roger-luo** [comment]
@GiggleLiu `add YaoBase#master` and commit Manifest here first? CI is not passing

File: test/register.jl [hunk 1]
```diff
@@ -170,3 +171,15 @@ end
     reg4.state[1] = 2.0
     @test reg.state[1] != 2.0
 end
+
+@testset "collapseto" begin
```
Comment (Roger-luo) [inline test/register.jl]: change the code below to `resetto!`

Comment (GiggleLiu) [reply test/register.jl]: This is the correct `collapseto`.

**Comment by Roger-luo** [comment]
This fails cuz doctest, seems fine.

**Pull request merged**